### ISSUE TITLE
What does "all instances are 100% verifiable" mean

### ISSUE THREAD

**Issue by neu-ma-tic** [pashword/pashword#10]
In the questions answered under the FAQ section, there is no mention of being verifiable or how that works, and the link leads to the deployments to vercel. If I deploy this to my own box, I wouldn't have the build logs publicly available, thus removing this sense of verifiability.

TL;DR: How do I verify that an instance of the app is what it says it is in an unmodified state?

**Comment by NayamAmarshe** [comment]
> In the questions answered under the FAQ section, there is no mention of being verifiable or how that works, and the link leads to the deployments to vercel. If I deploy this to my own box, I wouldn't have the build logs publicly available, thus removing this sense of verifiability.
> 
> TL;DR: How do I verify that an instance of the app is what it says it is in an unmodified state?

You can setup your own Github Actions workflow to run on every commit for this.

Pashword is verifiable because I cannot modify the code once it's been deployed, you can verify that by hosting your own Vercel instance as well.
